src/axom/inlet/Inlet.cpp

```diff
@@ -156,28 +162,30 @@ bool Inlet::get(const std::string& name, int& value)
 bool Inlet::get(const std::string& name, std::string& value)
 {
   axom::sidre::View* valueView = baseGet(name);
-  if (valueView == nullptr)
+  if(valueView == nullptr)
   {
     return false;
   }
 
-  if (valueView->getTypeID() != axom::sidre::CHAR8_STR_ID)
+  if(valueView->getTypeID() != axom::sidre::CHAR8_STR_ID)
   {
-    std::string msg = fmt::format("String named '{0}' was asked for"
-                                  " but recieved type {1}",
-                                  name, valueView->getTypeID());
+    std::string msg = fmt::format(
+      "String named '{0}' was asked for"
+      " but recieved type {1}",
+      name,
+      valueView->getTypeID());
```

Comment:
We might want to manually improve code like this, which was manually broken up to workaround limitations of our previous uncrustify settings,